cms/static/js/views/license.js

```diff
@@ -0,0 +1,139 @@
+define(["js/views/baseview", "underscore"], function(BaseView, _) {
+  var defaultLicenseInfo = {
+    "all-rights-reserved": {
+      "name": gettext("All Rights Reserved"),
+      "tooltip": gettext("You reserve all rights for your work")
+    },
+    "creative-commons": {
+      "name": gettext("Creative Commons"),
+      "tooltip": gettext("You waive some rights for your work, such that others can use it too"),
+      "url": "//creativecommons.org/about",
+      "options": {
+        "ver": {
+          "name": gettext("Version"),
+          "type": "string",
+          "default": "4.0",
+        },
+        "BY": {
+          "name": gettext("Attribution"),
+          "type": "boolean",
+          "default": true,
+          "help": gettext("Allow others to copy, distribute, display and perform " +
+            "your copyrighted work but only if they give credit the way you request."),
+          "disabled": true,
+        },
+        "NC": {
+          "name": gettext("Noncommercial"),
```

Comment:
Nit: should this be "Non-commercial" or is the non-hyphenated version equally valid?